src/js/projects.js

```diff
@@ -282,7 +282,7 @@ function ProjectsManager(dataManager, configurator, reporter, git, migrator) {
 
             if (meta.type.id === "tw") {
                 var dict = meta.project.id;
-                frames = dataManager.getAllWords(dict);
+                frames = dataManager.getAllWords(source.language_id, dict);
```

Comment:
@PurpleGuitar source was not defined in this context, we can update this in another PR